build error on main due to tokio 1.21.0 unstable feature

Build error just occurred on `main` when I merged #1328, probably unrelated?

```
err:    Compiling console-subscriber v0.1.7
err: error[E0308]: mismatched types
err:     --> /***/.cargo/registry/src/github.com-1ecc6299db9ec823/console-subscriber-0.1.7/src/lib.rs:1066:12
err:      |
err: 1061 | ) -> tokio::task::JoinHandle<T>
err:      |      -------------------------- expected `tokio::task::JoinHandle<T>` because of return type
err: ...
err: 1066 |     return tokio::task::Builder::new().name(_name).spawn(task);
err:      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `tokio::task::JoinHandle`, found enum `Result`
err:      |
err:      = note: expected struct `tokio::task::JoinHandle<_>`
err:                   found enum `Result<tokio::task::JoinHandle<_>, std::io::Error>`
err: 
err: For more information about this error, try `rustc --explain E0308`.
err: error: could not compile `console-subscriber` due to previous error
err: warning: build failed, waiting for other jobs to finish...
err: error: build failed
```

--- a/pd/Cargo.toml
+++ b/pd/Cargo.toml
@@ -50,7 +50,8 @@ bytes = "1"
 chrono = { version = "0.4", default-features = false, features = ["serde"] }
 csv = "1.1"
 directories = "4.0"
-tokio = { version = "1.16", features = ["full"]}
+# Temporarily pin due to https://github.com/penumbra-zone/penumbra/issues/1355
+tokio = { version = "=1.20.1", features = ["full"]}
 tokio-stream = "0.1"
 tokio-util = "0.7"
 tower = { version = "0.4", features = ["full"]}

